src/crypto/tls/handshake_messages_test.go

```diff
@@ -368,9 +368,11 @@ func (*certificateRequestMsgTLS13) Generate(rand *rand.Rand, size int) reflect.V
 	if rand.Intn(10) > 5 {
 		m.scts = true
 	}
+	m.supportDelegatedCredential = false
```

Comment:
Should we add in tests where these are true in various combinations?

Reply:
I agree on this. I'll incorporate.

Reply:
Added now.